test/unit/collision-circle.js

```diff
@@ -120,4 +120,170 @@ describe('CircleCollider', function() {
       expect(displacement.y / displacement.x).to.closeTo(2 / 6, MARGIN_OF_ERROR);
     });
   });
+
+  describe('updateFromSprite()', function() {
+    var pInst, testAnimation;
+
+    beforeEach(function() {
+      pInst = new p5(function() {});
+      testAnimation = createTestAnimation(3);
+    });
+
+    afterEach(function() {
+      pInst.remove();
+    });
+
+    it('adopts radius from animationless sprite when no radius is given, scale 1', function() {
+      var sprite = pInst.createSprite(0, 0, 200, 200);
+      sprite.setCollider('circle');
+      expect(sprite.collider.radius).to.equal(100);
+      expect(sprite.collider._scaledRadius).to.equal(100);
```

Comment:
`CircleCollider` stores unscaled size in `radius` and scaled size in `_scaledRadius`.